### DISCUSSION TITLE
website thread

### DISCUSSION THREAD

**Discussion by Ajinkgupta** [geekscommunity/.github#2] [General]
Everything about CoolCommunity X website Project reply to get started in this repository

**Comment by siiaaa** [comment]
![image](https://user-images.githubusercontent.com/121426629/209537281-3e8211bb-9aa4-4239-95f3-86e0fd9ff205.png)
 added navbar , footer , daisyui

**Reply by Ajinkgupta** [reply]
looking good

**Comment by Ajinkgupta** [comment]
> Everything about CoolCommunity X website Project reply to get started in this repository

upgraded to GeeksCommunity
